smdebug/core/hook.py

```diff
@@ -395,6 +398,12 @@ def has_default_hook_configuration(self):
             return True
         return False
 
+    def has_default_profiler_configuration(self):
```

Comment:
nit: if this function is only used internally (in `has_default_configuration()`), maybe prepend an underscore to the function name? Otherwise, an error handler decorator should be added to this function.

Reply:
Done.